sdk/attestation/azure-security-attestation/CHANGELOG.md

```diff
@@ -9,10 +9,6 @@
 - `AddIsolatedModeCertificatesOptions` becomes `AddIsolatedModeCertificateOptions`
 - `RemoveIsolatedModeCertificatesOptions` becomes `RemoveIsolatedModeCertificateOptions`
 - Renamed `AttestEnclaveOptions` to `AttestSgxEnclaveOptions` and `AttestOpenEnclaveOptions`.
-- Split out `AttestationClient::Create` into its own factory class `AttestationClientFactory`.
```

Comment:
Didn't we also remove `RetrieveResponseValidationCollateral` in the previous PR or was that always private? I don't see it being called in the CL breaking change list.

Reply:
Removed in the previous CR.

Reply:
Looks like it was always private. We might want to update the docs on public API that reference the method still, because the end user can't call it:

https://github.com/Azure/azure-sdk-for-cpp/search?q=RetrieveResponseValidationCollateral

https://github.com/Azure/azure-sdk-for-cpp/blob/0fd02674fee2b1fefd0608e02320d80376a42b6e/sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp#L334

Reply:
It was public for 1.0.0.beta-1, and was removed for either 1.0.0.beta-2 or 1.0.0.beta-3 (not sure which).

Could you show me the public API documentation for this method? I thought I had searched for all of them and removed them, but it's possible I missed a couple.

Reply:
There are a couple places (I linked the GitHub search of the codebase above), here's an example:
https://github.com/Azure/azure-sdk-for-cpp/blob/ebe084bfc67c08f6d5aaf857dd7874729d9f5908/sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp#L98-L105